R/available_forecasts.R

```diff
@@ -1,10 +1,10 @@
 #' @title Count Number of Available Forecasts
 #'
 #' @description
-#'
-#' Given a data set with forecasts, count the number of available forecasts
-#' for arbitrary grouping (e.g. the number of forecasts per model, or the
-#' number of forecasts per model and location).
+#' Given a data set with forecasts, count the number of available forecasts.
```

Comment:
To be in line with the general style of the package
```suggestion
#' Given a data set with forecasts, this function counts the number of available forecasts.
```